draft-scenarios/clash/dragon_valley.tex

```diff
@@ -0,0 +1,159 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{Dragon Valley}{\images/dragon.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} LAAMAKALA
+
+%\textit{A valley where ancient dragons protect secrets long forgotten. Fight for power, claim the land, or perish like those before you.}
+
+\textit{A valley of secrets long forgotten. Its heart belongs to the beasts of old, while its fringes lure the bold and the desperate. Fight for power, claim the land, or perish like those before you.}
+
+% \textit{The valley calls to those who seek adventure and riches beyond imagination. Its heart is guarded by Dragons older than time, its edges lined with settlements of those who dare to claim a foothold in this perilous land. But there is no safety here—only the promise of power for those who fight, and the certainty of death for those who falter.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This Scenario plays out over 13 Rounds
+%(or a shorter version over 9 Rounds).
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2 -- 4
+
+\textbf{Starting Resources:} 14 \svg{gold}, 4 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item 2 × Few \svgunit{bronze} Units of the highest recruitment cost
+  %\item A Pack of \svgunit{bronze} Units of the lowest recruitment cost
+  %\item Pack of \svgunit{bronze} of second highest recruitment cost
+  %\item A Pack of \svgunit{bronze} Units of the player's choice
+\end{itemize}
+
+\textbf{Town Buildings:}
+\begin{itemize}
+  \item \svgunit{bronze} Dwelling
+  \item Each player may build the \svg{building_special_tent} building with a discount of 4 \svg{gold} and 2 \svg{building_materials}. %"Before the start" - is included in "Player Setup - town buildings". Could be added "Before start each player may..."
+\end{itemize}
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles ($P$ stands for the number of players) and arrange them as shown in the Scenario map layout:
+
+\begin{itemize}
+  \item P × Starting (I) Map Tile
+  \item 3P × Far (II-III) Map Tiles - one per player must include a Settlement, and one per player must include a Trading Post
+  \item 2P × Near (IV-V) Map Tile
+  \item P × Center (VI-VII) Map Tiles (C5 is treated as a Settlement)
+\end{itemize}
+
+\textbf{Additionally,}\\
+\textbf{for 2--3 players:}
+\begin{itemize}
+  \item 1 × Center (VI-VII) Map Tile with a Dragon Utopia
+\end{itemize}
+
+\textbf{for 4 players:}
+\begin{itemize}
+  \item 2 × Center (VI-VII) Map Tiles with a Dragon Utopia
```

Comment:
Or, you could include the Dragon Utopia in the layout, and just mention that you need another one for 4 players.